new-dti-website/components/footer.tsx

```diff
@@ -51,7 +51,9 @@ const Footer: React.FC<FooterProps> = ({ theme }) => (
       theme === 'dark' ? 'bg-black text-neutral-50' : 'bg-[#F5F5F5] text-[#0C0404]'
     } w-full h-[146px] md:px-10 lg:px-[60px] flex justify-between items-center md:flex-row flex-col`}
   >
-    <div className="sm:w-[336px] sm:text-center md:text-lg md:w-[310px] md:text-left lg:w-full text-sm font-medium flex items-center ">
+    <div
+      className={`sm:w-[336px] sm:text-center md:text-lg md:w-[310px] md:text-left lg:w-full text-sm font-medium flex items-center ${theme === 'dark' ? 'text-gray-300' : 'text-black'}`}
+    >
```

Comment:
Regarding your comment [here](https://github.com/cornell-dti/idol/pull/741#issuecomment-2506848231), yeah for consistency let's just make the footer the same color (black) everywhere. 

Make sure that the text and icons on the footer are dimmer too
![Screenshot 2024-11-28 at 8 06 32 PM](https://github.com/user-attachments/assets/5026bd36-cded-46ec-895a-bf95c6c0118e)